tests/CountryPerformanceOnRisk.component.test.js

```diff
@@ -1,33 +1,156 @@
 import { CountryPerformanceOnRisk, CountrySelect }
-from '../src/components/CountryPerformanceOnRisk.js';
+from '../src/components/CountryPerformanceOnRisk';
 import React from 'react';
 import {shallow} from 'enzyme';
 import toJson from 'enzyme-to-json';
 
 
 describe('Components are working fine', () => {
+  function setup() {
+    let props = {
+      dispatch: jest.fn(),
+      cubeByRiskByCountry: {
+        1: {
```

Comment:
I think we planned a meaningful id here not just `1` and to have this with nested key i.e. we would have:

```
cubeByRiskByCountry: {
  "riskId": {
    "countryId": [
       ...
     ]
  }
}
```